helm-chart/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   repository: supervisions/itop
   pullPolicy: Always
-  tag: 3.0
+  tag: 3.0.2
```

Comment:
This is effectively the same. The current version of tag 3.0 is always the latest version 3.0.x